package.json

```diff
@@ -20,6 +20,8 @@
     "start-with-local-graphql-and-article-converter": "cross-env LOCAL_GRAPHQL_API=true RAZZLE_LOCAL_ARTICLE_CONVERTER=true razzle start",
     "start-with-local-article-converter": "cross-env RAZZLE_LOCAL_ARTICLE_CONVERTER=true yarn start",
     "start-prod": "cross-env NODE_ENV=production node build/server | bunyan",
+    "cy": "CYPRESS_USE_FIXTURES=true BABEL_ENV=test cypress open",
+    "cy-run": "CYPRESS_USE_FIXTURES=true BABEL_ENV=test cypress run",
```

Comment:
Vil du ha samme navn som i ed? For du har jo allerede e2e-use-fixtures

Reply:
Stemmer, vil ha samme navn som i ed og at de skal gjøre samme ting. Consistency er digg.